test/testdata/namer/ancestors.rb.symbol-table-raw.exp

```diff
@@ -1,6 +1,6 @@
   class <S <C <U <root>>> $1> < <S <C <U Object>> $1> ()
-    method <S <C <U <root>>> $1><N <U <static-init>> $3> (<blk>) @ Loc {file=test/testdata/namer/ancestors.rb start=2:1 end=15:4}
+    method <S <C <U <root>>> $1><N <U <static-init>> $108> (<blk>) @ Loc {file=test/testdata/namer/ancestors.rb start=2:1 end=15:4}
```

Comment:
Each time we add an RBI this number will go up. We might want to elide these numbers too.

Reply:
I don't (yet?) see why it has to go up.
We could potentially do changes to make this number more stable